services/core-api/app/api/mines/project_summary/models/project_summary.py

```diff
@@ -143,3 +146,14 @@ def delete(self, commit=True):
             self.mine_documents.remove(doc.mine_document)
             doc.mine_document.delete(False)
         super(ProjectSummary, self).delete(commit)
+
+    def send_project_summary_email_to_ministry(self, mine):
+        recipients = PROJECT_SUMMARY_EMAILS
+
+        subject = f'Project Summary Notification for {mine.mine_name}'
+        body = f'<p>{mine.mine_name} (Mine no: {mine.mine_no}) has submitted Project Summary data in MineSpace</p>'
```

Comment:
"Mine no" does not appear to be coming through on the test email. Also, do we need a period at the end of the line 154 paragraph(looking at the A/C)?

Reply:
![image](https://user-images.githubusercontent.com/13891487/145091752-1d076b2e-f44c-4ca5-809f-f2e5e29bad3c.png)